x-pack/legacy/plugins/infra/public/pages/logs/log_entry_categories/page_setup_content.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiSpacer, EuiSteps, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import React, { useMemo } from 'react';
+
+import {
+  createInitialConfigurationStep,
+  createProcessStep,
+  LogAnalysisSetupPage,
+  LogAnalysisSetupPageContent,
+  LogAnalysisSetupPageHeader,
+} from '../../../components/logging/log_analysis_setup';
+import { useTrackPageview } from '../../../hooks/use_track_metric';
+import { useLogEntryCategoriesSetup } from './use_log_entry_categories_setup';
+
+export const LogEntryCategoriesSetupContent: React.FunctionComponent = () => {
+  useTrackPageview({ app: 'infra_logs', path: 'log_entry_categories_setup' });
```

Comment:
Could this be a duplicate of the line below?

Reply:
The line below has a delay so we track both the initial visit and the presence on the page after some time. @jasonrhodes is that still the way we want to go?

Reply:
mmm, Do you know if the receiving end knows that the delay is there or does it treat both entries the same? My guess is that it would count two visits :/

Reply:
AFAIK the delay is part of the key under which the event is recorded. Again, @jasonrhodes might know more.

Reply:
Yes, let's leave this as is for now. If the user leaves the page before the delay amount expires, the 2nd track won't be counted. It was a (very rough) way to try to get a sense of whether users were staying on a page or just bouncing away quickly. It's not perfect by a long shot, but the telemetry WG will hopefully give us better tools to replace this with. :)

Reply:
And yes, as @weltenwort says, the delay value is included as part of the stored key, so they are disambiguated.